seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -166,26 +166,26 @@
         "// [\"NICKEL\", 2.05],",
         "// [\"DIME\", 3.10],",
         "// [\"QUARTER\", 4.25],",
-        "// [\"ONE\", 90.00],",
-        "// [\"FIVE\", 55.00],",
-        "// [\"TEN\", 20.00],",
-        "// [\"TWENTY\", 60.00],",
-        "// [\"ONE HUNDRED\", 100.00]]",
+        "// [\"ONE\", 90],",
+        "// [\"FIVE\", 55],",
+        "// [\"TEN\", 20],",
+        "// [\"TWENTY\", 60],",
+        "// [\"ONE HUNDRED\", 100]]",
         "",
-        "checkCashRegister(19.50, 20.00, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90.00], [\"FIVE\", 55.00], [\"TEN\", 20.00], [\"TWENTY\", 60.00], [\"ONE HUNDRED\", 100.00]]);"
+        "checkCashRegister(19.50, 20, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90], [\"FIVE\", 55], [\"TEN\", 20], [\"TWENTY\", 60], [\"ONE HUNDRED\", 100]]);"
```

Comment:
Remove all trailing zeros that comes after decimal

- `checkCashRegister(19.50,`
- `[\"DIME\", 3.10]`